trompace/queries/templates.py

```diff
@@ -0,0 +1,97 @@
+# Templates for generate GraphQL queries for mutations.
+
+# To be added EntryPoint, ControlAction, PropertyValueSpecification and Property
+from trompace.exceptions import UnsupportedLanguageException
+from . import make_parameters, MUTATION
+from ..constants import SUPPORTED_LANGUAGES
+
+
+
+
+def query_create(identifier: str, query_string: str, name=None, publisher=None, contributor=None, creator=None,
+                    source=None, subject=None, description=None, language=None, coverage=None, date=None,
```

Comment:
I see a few parameters here which aren't used in the function body. I'm not sure this is a sustainable way of creating these queries, as it means that each time we add a query for a new type we'll have to see what's not here and add it. This could easily make us miss or duplicate items.
How about moving this code up a level (e.g. to the `query_document` method). We could create a dict:
`query_params = {"name": document_name, "publisher": publisher}`, and then have a version of the `make_parameters` method which skips a field if its value is None.

Reply:
Done